File: CYLTimePicker/CYLTimePicker/ViewController.m
//
//  ViewController.m
//  CYLTimePicker
//
//  Created by 迟钰林 on 2017/6/22.
//  Copyright © 2017年 迟钰林. All rights reserved.
//

#import "ViewController.h"
#import "CYLTimePicker.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    CYLTimePicker *p = [[CYLTimePicker alloc] initWithFrame:CGRectMake(0, 100, self.view.bounds.size.width, 250)];
    [self.view addSubview:p];
    
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: MagnicView/MagnicView/ViewController.swift
//
//  ViewController.swift
//  MagnicView
//
//  Created by 迟钰林 on 2017/6/20.
//  Copyright © 2017年 迟钰林. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let mv = MagnicView.init(frame: self.view.bounds)
        self.view.addSubview(mv)
        
    }
}

